modules/network-observability-flowcollector-view.adoc

```diff
@@ -45,19 +45,10 @@ spec:
       limits:
         memory: 800Mi
     conversationEndTimeout: 10s
-    logTypes: FLOWS                            <3>
+    logTypes: Flows                            <3>
     conversationHeartbeatInterval: 30s
```

Comment:
I'm still waiting for https://github.com/netobserv/network-observability-operator/pull/467 to be merged here.
`conversationEndTimeout` and `conversationHeartbeatInterval` are part of the fields moving to `debug` sections.

See the [entire list in Jira](https://issues.redhat.com/browse/NETOBSERV-1092)

Reply:
ACK, will be on the lookout for this to merge and remove these fields. The API doc regeneration PR (yet to be opened) will have the fields.

Reply:
yeah, relying on the API autogenerated doc shoud be good enough here :+1:

Reply:
FYI the `debug` fields has been renamed to `advanced` in the PR above :eyes: